app/models/package_manager/nu_get.rb

```diff
@@ -29,14 +29,27 @@ def self.install_instructions(db_project, version = nil)
 
     def self.deprecation_info(db_project)
       info = latest_remote_version(db_project.name)
-      deprecation = info&.dig("items")&.first&.dig("items")&.first&.dig("catalogEntry", "deprecation")
+      catalog_entry = info&.dig("items")&.first&.dig("items")&.first&.dig("catalogEntry")
 
-      # alternate_package is not currently stored in DB but I wanted to record it because it seems useful
-      {
-        is_deprecated: deprecation.present?,
-        message: deprecation.present? ? deprecation["message"] : "",
-        alternate_package: deprecation.present? ? deprecation.dig("alternatePackage", "id") : nil,
-      }
+      is_deprecated = catalog_entry["deprecation"].present? || catalog_entry["listed"] == false
+
+      if is_deprecated
+        message = catalog_entry.dig("deprecation", "message") || "unlisted"
+        # alternate_package is not currently stored in DB but I wanted to record it because it seems useful
+        alternate_package = catalog_entry.dig("deprecation", "alternatePackage", "id")
+
+        {
+          is_deprecated: true,
+          message: message,
+          alternate_package: alternate_package,
```

Comment:
Do any of the packages in the tests below have an alternative package? If so, can you update the tests to show that this data gathering is working?

Reply:
Not sure, but that would be good to add!